src/content/docs/apm/agents/php-agent/other-integrations/magento-integration.mdx

```diff
@@ -0,0 +1,289 @@
+        * By integrating this infrastructure agent in your system, one can able to 
+        know the CPU usage,Memory usage,Storage usage,Disk usage and Network traffic etc.
+
+        </Step>
+
+    <Step>
+        ## Install our PHP agent
+
+        Install PHP agent to instrument your Magento application and get Application and Browser related Metrics like AJAX performance,Error Rate,Duration,Core web vitals and Throughput.
+        Follow php documentation: https://docs.newrelic.com/docs/apm/agents/php-agent/getting-started/php-agent-compatibility-requirements/ to learn more about.
+
+        From one.newrelic.com:
+
+        1. Click Add data
+        2. Click Application monitoring
+        3. Click on PHP card
+        4. Select an account
+        5. Click Begin installation
+        6. Click on On host
+        7. Copy the command and run it on your terminal
+
+        </Step>
+    
+    <Step>
+        ## Install our MySQL agent
+
+        Install MySQL agent to monitor database related metrics like Database Utilization and Connection like Slowest queries and active connections by node.
+        Follow MySQL documentation: https://docs.newrelic.com/install/mysql/  
+
+        From one.newrelic.com:
+
+        1. Click Add data
+        2. Click Infrastructure & OS
+        3. Click on MySQL card
+        4. Select an account
+        5. Click Begin installation
+        6. Copy the command and run it on your terminal
+
+        </Step>
+    
+    <Step>
+        ## Install our Apache agent
+
+        Install Apache agent to know more about TCP connections, DNS lookup , HTTPS, SSL, Server and Worker status.
+        Follow Apache documentation: https://docs.newrelic.com/docs/infrastructure/host-integrations/host-integrations-list/apache-monitoring-integration/
+
+        From one.newrelic.com:
+                    
+        1. Click Add data
+        2. Click Infrastructure & OS
+        3. Click on Apache card
+        4. Select an account
+        5. Click Begin installation
+        6. Copy the command and run it on your terminal
+
+        </Step>
+    
+
+    <Step>
+        ## Install our Elasticsearch 
+        
+        Install Elasticsearch agent , if one is using Magento version above 2.4, one needs to install an ElasticSearch application and agent to monitor that application.You can monitor metrics like Cluster status and Current shards.
+        docs:https://docs.newrelic.com/docs/infrastructure/host-integrations/host-integrations-list/elasticsearch/elasticsearch-integration/
+        Reference:https://magentip.com/install-magento-2-4-x-on-ubuntu-with-elasticsearch/
+
+        From one.newrelic.com:
+                    
+        1. Click Add data
+        2. Click Infrastructure & OS
+        3. Click on Elasticsearch card
+        4. Select an account
+        5. Click Begin installation
+        6. Copy the command and run it on your terminal
+
+        </Step>
+    
+
+    <Step>
+        ## Set up synthetic monitoring
```

Comment:
Is this an optional or required step?

Reply:
It's required in order to capture the server downtime.